java/pom.xml

```diff
@@ -0,0 +1,188 @@
+
+    <organization>
+        <name>Dylibso, Inc.</name>
+        <url>https://dylib.so</url>
+    </organization>
+
+    <developers>
+        <developer>
+            <name>The Extism Authors</name>
+            <email>oss@extism.org</email>
+            <roles>
+                <role>Maintainer</role>
+            </roles>
+            <organization>Dylibso, Inc.</organization>
+            <organizationUrl>https://dylib.so</organizationUrl>
+        </developer>
+    </developers>
+
+    <scm>
+        <connection>scm:git:git://github.com/extism/extism.git</connection>
+        <developerConnection>scm:git:ssh://git@github.com/extism/extism.git</developerConnection>
+        <url>https://github.com/extism/extism/java</url>
+        <tag>main</tag>
+    </scm>
+
+    <issueManagement>
+        <system>Github</system>
+        <url>https://github.com/extism/extism/issues</url>
+    </issueManagement>
+
+    <properties>
+        <java.version>17</java.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+        <!-- dependencies -->
+        <jna.version>5.12.1</jna.version>
+        <gson.version>2.10</gson.version>
+
+        <!-- testing -->
+        <junit-jupiter-engine.version>5.9.1</junit-jupiter-engine.version>
+        <assertj-core.version>3.23.1</assertj-core.version>
+
+        <!-- maven plugins -->
+        <maven-compiler-plugin.version>3.10.1</maven-compiler-plugin.version>
+        <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
+
+        <!-- jreleaser -->
+        <jreleaser.git.root.search>true</jreleaser.git.root.search>
+    </properties>
+
+    <profiles>
+      <profile>
+        <id>release</id>
+        <build>
+          <plugins>
+            <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-javadoc-plugin</artifactId>
+              <version>3.4.1</version>
+              <executions>
+                <execution>
+                  <id>attach-javadoc</id>
+                  <goals>
+                    <goal>jar</goal>
+                  </goals>
+                </execution>
+              </executions>
+            </plugin>
+            <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-source-plugin</artifactId>
+              <version>3.2.1</version>
+              <executions>
+                <execution>
+                  <id>attach-source</id>
+                  <goals>
+                    <goal>jar</goal>
+                  </goals>
+                </execution>
+              </executions>
+            </plugin>
+            <plugin>
+              <groupId>org.jreleaser</groupId>
+              <artifactId>jreleaser-maven-plugin</artifactId>
+              <version>1.3.1</version>
+              <configuration>
+                <jreleaser>
+                  <release>
+                    <github>
+                      <enabled>false</enabled>
```

Comment:
set `<skipRelease>true</skipRelease>` instead to keep the configured releaser and skip the creation of a Git release during the invocation of `jreleaser:release` or `jreleaser:full-release`. Invoking `jreleaser:deploy` only publishes JARs & POMs to Maven Central.

Reply:
This seemed to work for me. Thanks for the tip. FWIW i still had to set a token to a dummy value:

```
t extism: Execution default-cli of goal org.jreleaser:jreleaser-maven-plugin:1.3.1:deploy failed: JReleaser has not been properly configured.
[ERROR] github.token must not be blank. Configure a value using the Maven DSL, or define a JRELEASER_GITHUB_TOKEN environment variable, or define a key/value pair in /Users/ben/.jreleaser/config.yml with key = JRELEASER_GITHUB_TOKEN
```

Reply:
Yes, that's true. A token is still required but as you noted it could be a any non-empty value. You'd need a true Github Personal Access Token when interacting with GitHub for real.